native/cocos/bindings/jswrapper/v8/Object.cpp

```diff
@@ -743,18 +743,8 @@ bool Object::call(const ValueArray &args, Object *thisObject, Value *rval /* = n
     }
 
     v8::Local<v8::Context> context = se::ScriptEngine::getInstance()->_getContext();
-    #if CC_DEBUG
-    v8::TryCatch tryCatch(__isolate);
-    #endif
     v8::MaybeLocal<v8::Value> result = _obj.handle(__isolate)->CallAsFunction(context, thiz, static_cast<int>(argc), pArgv);
 
-    #if CC_DEBUG
-    if (tryCatch.HasCaught()) {
-        v8::String::Utf8Value stack(__isolate, tryCatch.StackTrace(__isolate->GetCurrentContext()).ToLocalChecked());
-        SE_REPORT_ERROR("Invoking function failed, %s", *stack);
-    }
-    #endif
```

Comment:
Don't try catch script errors in se::Object::call. Otherwise, jsb.onError & CocosApplication::handleException will not be invoked in debug mode.